source/_reference-g-s.html.md.erb

```diff
@@ -7316,6 +7362,550 @@ For details about sending update requests, see [Create Update Request](#create-u
+  | **exclude** (optional) -- a comma-separated list of elements to not include in the response.<br/><ul><li>**displayValue**: excludes displayValue info for cells, rows, and summary fields</li><li>**image**: excludes image attributes for cells and summary fields</li><li>**imageAltText**: excludes alt text for cells and summary fields</li></ul>
+<%= RETURNS %> | [SheetSummary object](#sheetsummary-object) containing an array of [SummaryField objects](#summaryfield-object)
+<%= SCOPE %> | READ_SHEETS
+
+### Get Summary Fields
+
+> Example request: get summary fields
+
+```shell
+curl https://api.smartsheet.com/2.0/sheets/{sheetId}/summary/fields \
+-H "Authorization: Bearer ll352u9jujauoqz4gstvsae05"
+```
+
+```javascript
+var options = {
+  sheetId: 1421228469708676
+};
+
+smartsheet.sheets.getSummaryFields(options)
+  .then(function(summary) {
+    console.log(summary);
+  })
+  .catch(function(error) {
+    console.log(error);
+  });
+```
+
+```csharp
+// Not yet implemented
+```
+
+```java
+// Not yet implemented
+```
+
+```python
+# Not yet implemented
+```
+
+```ruby
+smartsheet.sheets.summaries.get_fields(sheet_id: 1421228469708676)
+```
+
+> Example response
+
+```json
+{
+  "pageNumber": 1,
+  "pageSize": 100,
+  "totalPages": 1,
+  "totalCount": 2,
+  "data": [
+    {
+      "id": 5629723890335892,
+      "index": 0,
+      "title": "Owner",
+      "type": "CONTACT_LIST",
+      "locked": false,
+      "lockedForUser": false,
+      "objectValue": {
+        "objectType": "CONTACT",
+        "email": "jane.roe@smartsheet.com",
+        "name": "Jane Roe",
+        "imageId": "u!1!8ljfd7w9-aY!AxDzH0wWv1Y!y9VvAgUOFdg"
+      },
+      "displayValue": "Jane Roe",
+      "createdAt": "2018-07-19T22:00:43Z",
+      "modifiedAt": "2018-07-19T22:00:43Z"
+    }
+  ]
+}
+```
+
+`GET /sheets/{sheetId}/summary/fields`
+
+Returns object containing array of summary fields. Allows for pagination of results.
+
+     |     |
+-----|-----|
+<%= HEADERS %> | <%= HEADER_AUTH %>
+<%= PARAMETERS %> | **include** (optional) -- a comma-separated list of elements to include in the response.<br/><ul><li>**format**: includes format info for cells, rows, and summary fields</li><li>**writerInfo**: includes createdBy and modifiedBy attributes for rows and summary fields</li></ul>
+  | **exclude** (optional) -- a comma-separated list of elements to not include in the response.<br/><ul><li>**displayValue**: excludes displayValue info for cells, rows, and summary fields</li><li>**image**: excludes image attributes for cells and summary fields</li><li>**imageAltText**: excludes alt text for cells and summary fields</li></ul>
+  | <%= PAGING_PARAMETERS %>
+<%= RETURNS %> | [SheetSummary object](#sheetsummary-object) containing an array of [SummaryField objects](#summaryfield-object)
+<%= SCOPE %> | READ_SHEETS
```

Comment:
Is this the write scope for updating fields? :thinking:

Reply:
Its confusing in the layout of the erb file with the samples intermixed (I got confused too), but that line is the scope for the GET request. He has the scope for PUT listed as `ADMIN_SHEETS`. Those should be correct, yes?

Reply:
Ah. Yes. Okay. I see now; I was indeed confused. Yes, `GET` should be `READ_SHEETS` and the rest of the `/summary` endpoints are `ADMIN_SHEETS`